keyboards/melgeek/z70ultra_v1/info.json

```diff
@@ -0,0 +1,389 @@
+                {"matrix": [0, 14], "x":15,   "y":0},
+
+                {"matrix": [1, 0],  "x":0,     "y":1, "w":1.5},
+                {"matrix": [1, 1],  "x":1.5,   "y":1},
+                {"matrix": [1, 2],  "x":2.5,   "y":1},
+                {"matrix": [1, 3],  "x":3.5,   "y":1},
+                {"matrix": [1, 4],  "x":4.5,   "y":1},
+                {"matrix": [1, 5],  "x":5.5,   "y":1},
+                {"matrix": [1, 6],  "x":6.5,   "y":1},
+                {"matrix": [1, 7],  "x":7.5,   "y":1},
+                {"matrix": [1, 8],  "x":8.5,   "y":1},
+                {"matrix": [1, 9],  "x":9.5,   "y":1},
+                {"matrix": [1, 10], "x":10.5,  "y":1},
+                {"matrix": [1, 11], "x":11.5,  "y":1},
+                {"matrix": [1, 12], "x":12.5,  "y":1},
+                {"matrix": [1, 13], "x":13.5,  "y":1, "w":1.5},
+                {"matrix": [1, 14], "x":15,    "y":1},
+
+                {"matrix": [2, 0],  "x":0,     "y":2, "w":1.75},
+                {"matrix": [2, 1],  "x":1.75,  "y":2},
+                {"matrix": [2, 2],  "x":2.75,  "y":2},
+                {"matrix": [2, 3],  "x":3.75,  "y":2},
+                {"matrix": [2, 4],  "x":4.75,  "y":2},
+                {"matrix": [2, 5],  "x":5.75,  "y":2},
+                {"matrix": [2, 6],  "x":6.75,  "y":2},
+                {"matrix": [2, 7],  "x":7.75,  "y":2},
+                {"matrix": [2, 8],  "x":8.75,  "y":2},
+                {"matrix": [2, 9],  "x":9.75,  "y":2},
+                {"matrix": [2, 10], "x":10.75, "y":2},
+                {"matrix": [2, 11], "x":11.75, "y":2},
+                {"matrix": [2, 13], "x":12.75, "y":2, "w":2.25},
+                {"matrix": [3, 14], "x":15,    "y":2},
+
+                {"matrix": [3, 0],  "x":0,     "y":3, "w":2.25},
+                {"matrix": [3, 2],  "x":2.25,  "y":3},
+                {"matrix": [3, 3],  "x":3.25,  "y":3},
+                {"matrix": [3, 4],  "x":4.25,  "y":3},
+                {"matrix": [3, 5],  "x":5.25,  "y":3},
+                {"matrix": [3, 6],  "x":6.25,  "y":3},
+                {"matrix": [3, 7],  "x":7.25,  "y":3},
+                {"matrix": [3, 8],  "x":8.25,  "y":3},
+                {"matrix": [3, 9],  "x":9.25,  "y":3},
+                {"matrix": [3, 10], "x":10.25, "y":3},
+                {"matrix": [3, 11], "x":11.25, "y":3},
+                {"matrix": [3, 12], "x":12.25, "y":3, "w":1.75},
+                {"matrix": [2, 14], "x":14,    "y":3},
+                {"matrix": [4, 14], "x":15,    "y":3},
+
+                {"matrix": [4, 0],  "x":0,     "y":4, "w":1.25},
+                {"matrix": [4, 1],  "x":1.25,  "y":4, "w":1.25},
+                {"matrix": [4, 2],  "x":2.5,   "y":4, "w":1.25},
+                {"matrix": [4, 4],  "x":3.75,  "y":4, "w":3},
+                {"matrix": [4, 5],  "x":3.75,  "y":4, "w":6.25},
+                {"matrix": [4, 6],  "x":6.75,  "y":4, "w":3},
+                {"matrix": [4, 9],  "x":9.75,  "y":4, "w":1.25},
```

Comment:
Fixes some key overlap issues in QMK Configurator.

![image](https://github.com/qmk/qmk_firmware/assets/18669334/d0bf1017-4831-4580-8d60-6c005cc5d2a5)

```suggestion
                {"matrix": [4, 4],  "x":3.75,  "y":4.5, "w":3, "h": 0.5},
                {"matrix": [4, 5],  "x":3.75,  "y":4, "w":6, "h": 0.5},
                {"matrix": [4, 6],  "x":6.75,  "y":4.5, "w":3, "h": 0.5},
                {"matrix": [4, 9],  "x":9.75,  "y":4, "w":1.5},
```